core/config/project_settings.cpp

```diff
@@ -1132,12 +1132,14 @@ Variant ProjectSettings::get_setting(const String &p_setting, const Variant &p_d
 	}
 }
 
+#define GLOBAL_CLASS_LIST_CACHE_FILE "global_script_class_cache.cfg"
+
 Array ProjectSettings::get_global_class_list() {
 	Array script_classes;
 
 	Ref<ConfigFile> cf;
 	cf.instantiate();
-	if (cf->load(get_project_data_path().path_join("global_script_class_cache.cfg")) == OK) {
+	if (cf->load(get_project_data_path().path_join(GLOBAL_CLASS_LIST_CACHE_FILE)) == OK) {
```

Comment:
I would use `get_global_class_list_path()` here, and in `store_global_class_list()` too.

Then there's maybe not need for the define anymore as it will be hardcoded only once.